méli-mélo/2024-04-stepsquiz/js/stepsquiz.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @title WET-BOEW Steps Quiz plugin
+ * @overview Quiz pattern for use with the Steps Form plugin.
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author ricokola
+ */
+( function( $, document ) {
+"use strict";
+
+//Detect the enhancement of the quiz
+var quizSelector = ".wb-steps.quiz",
```

Comment:
Considering that your CSS is more specific, let be as specific as the CSS selector

```suggestion
var quizSelector = ".provisional.wb-steps.quiz",
```